CardScan/Classes/CreditCardOcr/OcrMainLoop.swift

```diff
@@ -260,7 +260,7 @@ public class OcrMainLoop : MachineLearningLoop {
     }
     
     @objc func didBecomeActive() {
-        inBackground = false
+        mutexQueue.sync { self.inBackground = false }
         for queue in machineLearningQueues {
```

Comment:
should this be in the mutexQueue?

Reply:
it's just to make sure that all shared state is accessed from the `mutexQueue` note that we're using `sync` here